File: node/1.js
const AWS = require('aws-sdk');
const params = {Bucket: 'ec-lamba-test', Key: 'fadetoblack.txt'};
var returndata = "";

exports.handler = (event, context, callback) => {
    var s3 = new AWS.S3({apiVersion: '2006-03-01'});
    
    s3.getObject(params, function(err, data) {
    if (err) console.log(err, err.stack); // an error occurred
     else {
         returndata = Buffer.from(data.Body).toString('utf8');
         const response = {
            statusCode: 200,
            body: returndata
        };
    callback(null, response);
         }// successful response
    });
    
};

File: python/1.py
import json
import boto3

def lambda_handler(event, context):
    # TODO implement

    client = boto3.client('s3')
    s3 = boto3.client('s3')
    data = s3.get_object(Bucket='ec-lamba-test', Key='fadetoblack.txt')
    contents = data['Body'].read().decode('utf-8') 
    return contents
    #print(contents)


